eks-anywhere-common/Addons/Partner/Pulumi/pulumi-access-token-externalsecret.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: pulumi-access-token
+  namespace: pulumi
+spec:
+  refreshInterval: 1m
+  secretStoreRef:
+    name: eksa-secret-store #The secret store name we have just created.
+    kind: ClusterSecretStore
+  target:
+    name: pulumi-access-token # Secret name in k8s
+  data:
+    - secretKey: pulumi-access-token # which key it's going to be stored
+      remoteRef:
+        key: value # Our secret-name goes here
```

Comment:
I would rather have a specific name for the KVP than `value`

Reply:
`secretKey` should be a something reasonable like `accessToken` and key should the keyname in secrets manager which is `pulumi-access-token`